src/libraries/StringParser.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {VmSafe} from "forge-std/Vm.sol";
+import {LibString} from "@solady/utils/LibString.sol";
+
+/// @title StringParser
+/// @notice Library for parsing string representations of decimal numbers
+library StringParser {
```

Comment:
I'm not sure about this library name, it's v generic when this logic is pretty specific to parsing decimal amounts to then later be normalized.

Reply:
Agreed with your above comment that we should just merge the two into one library